homeassistant/components/august/manifest.json

```diff
@@ -28,5 +28,5 @@
   "documentation": "https://www.home-assistant.io/integrations/august",
   "iot_class": "cloud_push",
   "loggers": ["pubnub", "yalexs"],
-  "requirements": ["yalexs==2.0.0", "yalexs-ble==2.4.2"]
+  "requirements": ["yalexs==3.0.0", "yalexs-ble==2.4.2"]
```

Comment:
is this PR depending on this bump? Otherwise please do it in a separate PR

Reply:
Usually we ask it to be split, but only when there are no breaking changes.

It ok to bump in the same PR as there are breaking changes as there are new lock states that need to be handled